express-static/index.js

```diff
@@ -0,0 +1,15 @@
+// for server code
+const express = require('express');
+const path = require('path');
+const app = express();
+
+const newPath = path.join(__dirname, 'public');
+console.log(newPath);
+
+const theStatic = express.static('public');
```

Comment:
Here you would pass your absolute path variable as the argument to `express.static()`